src/charm.py

```diff
@@ -154,6 +154,7 @@ def __init__(self, *args):
         self.ingress = IngressPerAppRequirer(
             self,
             relation_name="ingress",
+            strip_prefix=True,
```

Comment:
What prefix is this stripping? I don't understand how it just says "True"

Reply:
in traefik the default is `routing_mode=path` and the standard is `<model>-<application>`

so anything will be exposed as `<external-domain>/<model>-<application>` so u have to strip it due to the rewrites